docs/content/tutorial/5-group-stack.md

```diff
@@ -1,41 +1,72 @@
 ## Group/stack
 
-The following example shows how to group and stack elements of a bar chart.
+The following example shows how to group, regroup and stack elements of a bar chart.
 
-To get a stacked chart, you need to add a new dimension to the same channel 
-where the measure is: the y-axis. However, since doing only this 
-would result in multiple column chart elements with the same color stacked on
-top of each other, we also add the same dimension to the color channel.
+To get a grouped chart, you need to add a new dimension to the channel where you
+would like the data to be grouped: the x-axis. However, since doing only this 
+would result in several columns with the same color next to each other, 
+we also add the same dimension to the color channel.
 
-```javascript { "title": "Creating a stacked chart" }
+```javascript { "title": "Creating a grouped chart" }
 chart.animate({
 	config: {
 		channels: {
-			y: { attach: ['Kinds'] },
-			color: { attach: ['Kinds'] }
+			x: { attach: ['Genres'] },
+			color: { attach: ['Genres'] }
 		}
 	}
 })
 ```
 
-By detaching this newly added dimension from the y-axis and attaching it to the 
-x-axis, you get a grouped bar chart in a way that is easy to follow for the viewer.
+To change the grouping of the chart, you just need to put replace the recently
```

Comment:
In order to change the category via which the elements are grouped, just replace the dimension with another one on the same axis.